cmd/plume/upload.go

```diff
@@ -50,8 +51,8 @@ var (
 func init() {
 	cmdUpload.Flags.StringVar(&bucket, "bucket", "gs://coreos-plume", "gs://bucket/prefix/")
```

Comment:
For reference the existing default for `--upload` in build_image and similar is: `gs://users.developer.core-os.net/$USER/boards/$BOARD/$VERSION` which may be a useful pattern to stick with.